_includes/header.html

```diff
@@ -105,24 +107,23 @@
               </li>
             </ul>
           </div>
-          <div class="module widget-handle language left">
+          <div class="module widget-handle language left" style="padding: 0 0 !important;">
```

Comment:
Do not add inline CSS, use class

Reply:
without !important tag it is not going to work because i m ovewriting the already defined classes and their values if do not use that it will show default google plugin style

Reply:
coming to inline tag i will fix it

anything other than this
@fragm3

Reply:
@subhahu123  Then you should search for the class in CSS and add property there.

Reply:
@sk9331657  for what

Reply:
@subhahu123 please eliminate !important. You can do that by using more specific selectors in CSS. Read more here: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

Reply:
ok i will do that anything other than that ?
@fragm3

Reply:
Read my review comment on top, that would be the changes you need to make.

Reply:
![screenshot from 2018-11-16 01-09-15](https://user-images.githubusercontent.com/34541684/48577087-45040100-e93c-11e8-9b55-17a9ee8fc424.png)
@fragm3 
i have checked your given link but found nothing i have tried everything but google (pre)defined css always have more weightage  . i have tried everthing when website loads google translator plugin javascript it use it defined css but using !important in css/gtranslator.css file it's not going to look good for our UI 

comiing to this one which u mentioned above
 <div class="module widget-handle language left">
          <div class="module widget-handle language left" style="padding: 0 0 !important;">

i can remove !important from here and even the whole style 
but not in gtranslator.css
about indentation i will do that

Reply:
@subhahu123 ok, and do resolve the conflicts.

Reply:
@fragm3  
sorry but what ???????????????resolve the conflicts.

Reply:
@fragm3 
This has become very complicated 
if u think these changes are fine and ready for merge 
then i can open new PR with squashed and resolved 

but correct this one is like a....hell... 
i have tried very much but this has become very complicated because of various pr in my own repo

so if these changes are ..
then only there is a point to correct this .
so, let me know

Reply:
@fragm3  waiting for  reply

Reply:
@subhahu123 yes create a new PR, the changes are correct.

Reply:
done @fragm3